src/main/core/manager.c

```diff
@@ -274,6 +276,10 @@ Manager* manager_new(Controller* controller, ConfigOptions* config, SimulationTi
     /* now make sure the hosts path exists, as it may not have been in the template */
     g_mkdir_with_parents(manager->hostsPath, 0775);
 
+    if (config_getProgress(config) && isatty(STDERR_FILENO) == 1) {
```

Comment:
> Warn if a status bar was requested but we detected a non-tty?

I added a warning for this case.

> For non-ttys it'd still be useful to just periodically print the current status. Maybe that's a feature to add later though.

I agree, but I'll leave this for the future since I'm not sure the best way to do this.